opentreemap/api/tests.py

```diff
@@ -2003,26 +2003,26 @@ def test_401(self):
         self.assertEqual(ret.status_code, 401)
 
     def test_ok(self):
-        auth = base64.b64encode("jim:password")
+        auth = base64.b64encode(b"jim:password")
         withauth = {"HTTP_AUTHORIZATION": "Basic %s" % auth}
-
         ret = get_signed(self.client, "%s/user" % API_PFX, **withauth)
         self.assertEqual(ret.status_code, 200)
 
     def test_malformed_auth(self):
-        withauth = {"HTTP_AUTHORIZATION": "FUUBAR"}
+        auth = base64.b64encode(b"FUUBAR")
```

Comment:
I am just finding out in later test fixing this should be `auth = base64.b64encode(b"FUUBAR").decode()` and likewise of the changes in this test class. While `base64.b64encode()` now requires bytes, the request headers want strings

Reply:
eh standby, false positive.

Reply:
OK i still think this is true even though these tests still fail.